pyriemann/utils/tangentspace.py

```diff
@@ -621,27 +620,31 @@ def untangent_space(T, Cref, *, metric="riemann"):
 
 ###############################################################################
 
-
 # NOT IN API
-def transport(Covs, Cref, metric="riemann"):
-    """Parallel transport of a set of SPD matrices towards a reference matrix.
+def transport(X, C1, C2):
+    r"""Parallel transport of a set of SPD matrices towards a reference matrix.
+
+    The parallel transport of a set of SPD matrices :math:`\mathbf{X}_i`
+    from a reference matrix :math:`\mathbf{C}_1` towards a target matrix
+    :math:`\mathbf{C}_2` according to the Levi-Civita connection along
+    a geodesic under the affine invariant metric.
```

Comment:
Can you add a reference?

Reply:
The changes are already significant enough that I suggest going all the way.

For me, the documentation needs to be fixed and it needs to be made clear that input matrices are in the tangent space, not the manifold.